tests/unit_tests/ensemble_evaluator/ensemble_evaluator_utils.py

```diff
@@ -194,11 +184,11 @@ def evaluate(self, config):
     def start(self):
         self._eval_thread.start()
 
-    def _shouldFailJob(self, real, step, job):
-        return (real, 0, step, job) in self.fail_jobs
+    def _shouldFailJob(self, real, job):
+        return (real, 0, job) in self.fail_jobs
```

Comment:
What is the second parameter `-` - can we remove it?

Reply:
Thats probably a step that we should remove. Nice catch!